spec.md

```diff
@@ -3478,11 +3502,11 @@ linecol = digit_dec , { digit_dec } , ":" , digit_dec , { digit_dec } ;
 info = "@" , "[" , { string , " " , linecol } , "]" ;
 
 (* Type definitions *)
-width = "<" , int , ">" ;
+width = "<" , int_any , ">" ;
```

Comment:
Can we just not allow this?

Reply:
I would like to! I'm staging a follow-on major change to do this.